lib/list.gd

```diff
@@ -1002,6 +1002,34 @@ DeclareOperation( "PositionsProperty", [ IsList, IsFunction ] );
 DeclareOperation( "PositionBound", [ IsList ] );
 
 
+#############################################################################
+##
+#O  PositionsBound( <list> ) . . . . position of first bound element in a list
+##
+##  <#GAPDoc Label="PositionsBound">
+##  <ManSection>
+##  <Oper Name="PositionsBound" Arg='list'/>
+##
+##  <Description>
+##  returns the set of all indices for which an element is bound in the list
```

Comment:
This sounds slightly off to me... perhaps "returns the set of all indices of the list which are  bound to an element" ?

Reply:
I oriented myself on the documentation of PositionProperty just above. Should I change that as well (if I'm already at it)?

Reply:
Sure. Though we might also want to get another opinion by a native speaker like @ChrisJefferson or @stevelinton

Reply:
I'm not sure which is better. I don't really like "indices which are bound" since it isn't really the indices which are bound, but the corresponding list positions. "indices for which an element is bound in the list" is a little convoluted, but precise.  Neither is genuinely likely to be confusing.

Reply:
How would you like "the set of all bound positions of the list"?

Reply:
Done.